tests/e2e/specs/subscriptions/shopper/shopper-subscriptions-purchase-no-sign-up-fee.spec.js

```diff
@@ -2,50 +2,36 @@
  * External dependencies
  */
 import config from 'config';
-
 const { merchant, shopper, withRestApi } = require( '@woocommerce/e2e-utils' );
-
 import {
 	RUN_SUBSCRIPTIONS_TESTS,
 	describeif,
 	merchantWCP,
 	uiLoaded,
 } from '../../../utils';
-
 import { fillCardDetails, setupCheckout } from '../../../utils/payments';
 
-const productName = 'Subscription free trial product';
-const productSlug = 'subscription-free-trial-product';
-
-const customerBilling = config.get( 'addresses.customer.billing' );
-
+const productName = 'Subscription no signup fee product';
+const productSlug = 'subscription-no-signup-fee-product';
+const customerBilling = config.get(
+	'addresses.subscriptions-customer.billing'
+);
 let orderId;
 
 describeif( RUN_SUBSCRIPTIONS_TESTS )(
 	'Subscriptions > Purchase subscription without signup fee (free trial)',
 	() => {
 		beforeAll( async () => {
-			await merchant.login();
-
-			// Create subscription product without signup fee
-			await merchantWCP.createSubscriptionProduct(
-				productName,
-				'month',
-				false
-			);
-
-			await merchant.logout();
+			// Delete the user, if present
+			await withRestApi.deleteCustomerByEmail( customerBilling.email );
```

Comment:
Why do we need to do this both before and after all?

Reply:
Originally the user was deleted after each test suite run. However I added it to delete the user before the test to make sure that the user account isn't present causing test failure. Though this is a rare case.

Reply:
If we keep it in the `beforeAll` of all tests then logically we shouldn't need it in the `afterAll` as that becomes repetitive when the test suite runs, so we can keep it consistent imo.

Reply:
Sure. Updated